settings/trex.toml

```diff
@@ -13,6 +13,11 @@ predefined = "web_mercator"
 
 [[tileset]]
 name = "vapour_trail"
+attribution = "© Jungle Bus"
```

Comment:
we should also credit OSM contributors.
The readme suggests [Jungle Bus](http://junglebus.io/) [© OpenStreetMap contributors](http://www.openstreetmap.org/copyright)